build.gradle

```diff
@@ -23,6 +24,10 @@ repositories {
 	mavenCentral()
 }
 
+checkstyle {
+	toolVersion = "8.16"
+  configFile = new File(rootDir, "checkstyle.xml")
```

Comment:
@JStonham This is the one! Can you tell what the difference is between this line and the one above? What if you open it in IntelliJ, can you see a difference then?

Reply:
Oh!! I confused myself! I was thinking it had to be related to brackets, but you didn't suggest that - I did!